r/R/parquet.R

```diff
@@ -52,10 +52,16 @@ read_parquet <- function(file,
     schema <- reader$GetSchema()
     names <- names(schema)
     indices <- match(vars_select(names, !!col_select), names) - 1L
-    tab <- reader$ReadTable(indices)
+    tab <- tryCatch(
+      reader$ReadTable(indices),
+      error = function(e) { read_compressed_error(e) }
```

Comment:
```suggestion
      error = read_compressed_error
```

Reply:
I've applied the same change to all similar lines in this PR